### PR TITLE
Simplifying comment templates

### PR DESCRIPTION
**Pull request by NoLogo**

### CODE REVIEW

File: groups/templates/groups/text_comment.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    {{ comment.user }} wrote at <a href="{{ comment.get_pagejump }}">{{ comment.date_created }}</a>:
+    {% if comment.user_may_delete %}(<a href="{% url 'comment-delete' pk=comment.pk %}">delete this comment</a>){% endif %}
+</li>
+<li>
+    {% if comment.is_deleted %}
+        (Post deleted)
+    {% else %}
+        <p>{{ comment.body }}</p>
+    {% endif %}
+</li>
```
Comment (meshy) [inline groups/templates/groups/text_comment.html]: These templates could be dried into `groups/comment_base.html`

**Comment by NoLogo** [comment]
bump

File: groups/templates/groups/comment_base.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% block comment_header %}
+<li>
+    {{ comment.user }} wrote at <a href="{{ comment.get_pagejump }}">{{ comment.date_created }}</a>:
+    {% if comment.user_may_delete %}(<a href="{% url 'comment-delete' pk=comment.pk %}">delete this comment</a>){% endif %}
+</li>
```
Comment (meshy) [inline groups/templates/groups/comment_base.html]: Needs more indentation, please!

File: groups/templates/groups/comment_base.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    {% if comment.is_deleted %}
+        {% block comment_deleted %}
+            (Post deleted)
+        {% endblock comment_deleted %}
+    {% else %}
+        {% block comment_visible %}
+        {% endblock comment_visible %}
+    {% endif %}
+</li>
+{% endblock comment_body %}
```
Comment (meshy) [inline groups/templates/groups/comment_base.html]: And here!

**Comment by meshy** [comment]
Looks good to me, just needs some template love.

**Comment by NoLogo** [comment]
Bump

**Comment by meshy** [comment]
:+1:

**Pull request merged**